### ISSUE TITLE
Why the SegNet_ConvLSTM outputs F.log_softmax(up1, dim=1)?

### ISSUE THREAD

**Issue by somebodyus** [qinnzou/Robust-Lane-Detection#16]
Why the `SegNet_ConvLSTM` and `SegNet` output `F.log_softmax(up1, dim=1)`?
The Cross Entropy loss function has already the softmax operation.
